jupyterbook/content/Other Resources/ioos_installation_conda.md

```diff
@@ -0,0 +1,129 @@
+# Installing the IOOS conda environment
+For IOOS python users we recommend the free
+[Miniconda](http://conda.pydata.org/miniconda.html) Python distribution,
+a lightweight version of the [Anaconda Scientific Python Distribution](https://store.continuum.io/cshop/anaconda/).
+While the full Anaconda distribution will also work,
+it's faster to install Miniconda,
+and you install only the packages you need.
+If for some reason you decide later that you want the full Anaconda distribution,
+you can install it by typing `conda install anaconda` using miniconda.
+
+## Install
+
+Download and install the appropriate Miniconda installer from [http://conda.pydata.org/miniconda.html](http://conda.pydata.org/miniconda.html).
+With Anaconda you can create environments that use any Python version (e.g. Python 2.7 or Python 3.6),
```

Comment:
The 2.7 and 3.6 in the text show how dated these instructions are :-/
I'll update them ASAP!